R/utils.R

```diff
@@ -40,8 +40,8 @@ modify_list <- function(.x, ..., error_call = caller_env()) {
 }
 
 
-sys_sleep <- function(seconds, fps = 10, error_call = caller_env()) {
-  check_number_decimal(seconds, call = error_call)
+sys_sleep <- function(seconds, task, fps = 10) {
```

Comment:
@mgirlich I reverted the use of `error_call` here because I don't think it's useful as `seconds` is not directly passed to `sys_sleep()`. This is more of an internal fallback to ensure that `seconds` is the right type; validation of user inputs should happen earlier.